internal/restapi/v0/integrations/get.go

```diff
@@ -0,0 +1,70 @@
+package integrations
+
+import (
+	"context"
+	"errors"
+	"github.com/google/uuid"
+	"github.com/shinobistack/gokakashi/ent"
+	"github.com/swaggest/usecase/status"
+)
+
+type GetIntegrationRequests struct {
+	ID string `path:"id"`
+}
+
+type GetIntegrationResponse struct {
+	ID     string                 `json:"id"`
+	Name   string                 `json:"name"`
+	Type   string                 `json:"type"`
+	Config map[string]interface{} `json:"config"`
+}
+
+type ListIntegrationResponse struct {
+	Integrations []GetIntegrationResponse `json:"integrations"`
+}
+
+func GetIntegration(client *ent.Client) func(ctx context.Context, req GetIntegrationRequests, res *GetIntegrationResponse) error {
+	return func(ctx context.Context, req GetIntegrationRequests, res *GetIntegrationResponse) error {
+
```

Comment:
nit: avoid empty line feed at the start of the function.

(I think the linter might warn about it)